README.md

```diff
@@ -721,6 +724,119 @@ async function main() {
 main();
 ```
 
+## Downscoped Client
+
+[Downscoping with Credential Access Boundaries](https://cloud.google.com/iam/docs/downscoping-short-lived-credentials) is used to restrict the Identity and Access Management (IAM) permissions that a short-lived credential can use.
+
+The `DownscopedClient` class can be used to produce a downscoped access token from a 
+`CredentialAccessBoundary` and a source credential. The Credential Access Boundary specifies which resources the newly created credential can access, as well as an upper bound on the permissions that are available on each resource. Using downscoped credentials ensures tokens in flight always have the least privileges, e.g. Principle of Least Privilege.
+
+Notice: 
+Only Cloud Storage supports Credential Access Boundaries for now.
```

Comment:
Are we tracking work to add support for additional clients in the future?

Reply:
We don't currently have any plans to support additional clients right now as far as I know.

Reply:
We don't currently have any plans to support additional clients right now as far as I know.